CHANGELOG.md

```diff
@@ -559,9 +561,6 @@ The next release will require at least [Go 1.21].
 
 - Change interceptors in `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` to disable `SENT`/`RECEIVED` events.
   Use `WithMessageEvents()` to turn back on. (#3964)
-
-### Changed
-
```

Comment:
This shouldn't be removed, right?

Reply:
I removed it based on this bullet point in issue #7006 

- ./CHANGELOG.md:553 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Changed"]

(The line numbers have changed since the issue was first created.)

Reply:
Ah, I see. Thanks :+1: